azure-devops/projects/pagopa-projects/tlscert-prod-api-prod-platform-pagopa-it.tf

```
@@ -0,0 +1,75 @@
+variable "tlscert-prod-api-prod-platform-pagopa-it" {
```

```suggestion
variable "tlscert-prod-api-platform-pagopa-it" {
```
please change also file name
`tlscert-ENV-DNS-RECORD.tf`

for PROD DNS-RECORD is `api-platform-pagopa-it` (replace `.` with `-`)